core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/node-exporter.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Node Exporter | Populate service facts
+  service_facts:
+  when: ansible_facts.services is undefined
+
+- name: Node Exporter | Include defaults from node_exporter role
+  include_vars:
+    file: roles/node_exporter/defaults/main.yml
+
+- name: Node Exporter | Include specification vars from node_exporter role
+  include_vars:
+    file: roles/node_exporter/vars/manifest.yml
+
+- name: Node Exporter | Test if node exporter service is configured
+  assert:
+    that: "'{{ exporter.service.name }}.service' in ansible_facts.services"
+    fail_msg: "{{ exporter.service.name }} service is not found"
+    quiet: true
+
+- name: Node Exporter | Set facts
+  set_fact:
+    exporter_file_name: "node_exporter-{{ exporter.version }}.linux-amd64.tar.gz"
+
+- name: Node Exporter | Collect version
+  shell: |-
```

Comment:
Im not a proponent of using shell(instead of command) module for simple oneliners.